.cirrus.yml

```diff
@@ -170,7 +171,15 @@ ruling_win_task:
   depends_on:
     - build
   <<: *ONLY_SONARSOURCE_QA
-  <<: *WINDOWS_VM_DEFINITION
+  gce_instance:
+    image_project: language-team
+    image_family: lt-base-windows-jdk17
+    platform: windows
+    zone: us-central1-a
+    preemptible: false
+    disk: 128
+    use_ssd: true
+    type: n2d-standard-4
```

Comment:
You removed the last reference to `&WINDOWS_VM_DEFINITION` without removing the definition of it. And now we have two duplicated `gce_instance` blocks. Instead, I suggest keeping `&WINDOWS_VM_DEFINITION` definition, changing `image_family: lt-base-windows-jdk17` and `type: n2d-standard-4`, and using the reference 2 times, for `qa_os_win_task` and `ruling_win_task`.